### PR TITLE
Standardized the assets

### LINKED ISSUE #1441: Standardize the location of repository assets
**Describe the bug**

1. Our typescript repos store image and video assets in different locations.
2. This needs to be standardized as we may have unwanted images and videos uploaded to the repo

**To Reproduce**
You can see that these assets are stored in the root directory
- https://github.com/PalisadoesFoundation/talawa-api

**Expected behavior**

1. Standardize on the `src/assets` directory structure used in Talawa Admin
    1. https://github.com/PalisadoesFoundation/talawa-admin/tree/main/src/assets
1. Move the files referenced in the root directory markdown files to the appropriate sub directory under the assets directory
1. Update the markdown files to reference these new locations 
1. Delete any of these assets files not referenced in the markdown files, but verify that they are not referenced in the code base in any of the other Talawa repos first. If so propose a methodology to make sure these repos' functionality is maintained.
1. The code base mutations may be uploading files to the `video/` and `image/` directories. If this is the case
    1.  Make sure that files are never uploaded to these directories in the code base (This can be assured by configuring the `.gitignore` file)
    1.  Make sure that the directories remain. (This can be assured by configuring the `.gitignore` file)
1. Make sure no objects in the markdown files refer to these locations and if so move them to the new directory structure.
     1. other people's repos

**Actual behavior**
- See Above

**Screenshots**
- N/A

**Additional details**
- N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Veer0x1**
**What kind of change does this PR introduce?**
Standardized the assets such as images and videos in accordance to Talawa-Admin repo.

**Issue Number:**

Fixes #1441 

**Did you add tests for your changes?**
NA

**Snapshots/Videos:**
Screenshot 2024-01-03 at 6 06 57 PM

**If relevant, did you update the documentation?**

**Summary**
Moved all the images used in mardown files to `public/markdown/images` and deleted unused images and videos. And git ignored the `image` and `videos` directory and `public/markdown/*` and fixed the breaking test.

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by Veer0x1** [comment]
@palisadoes review, please.

File: .gitignore [hunk 1]
```diff
@@ -10,7 +10,9 @@ docs/schema.json
 node_modules
 .idea
 .env
-images
+image/*
+videos/*
+public/markdown/*
```
Comment (palisadoes) [inline .gitignore:15]: Why was this included?

File: .gitignore [hunk 2]
```diff
@@ -10,7 +10,9 @@ docs/schema.json
 node_modules
 .idea
 .env
-images
+image/*
```
Comment (palisadoes) [inline .gitignore:13]: Ensure that:

1. any subdirectories in this directory also gets ignored
2. it only affects the directory in root

File: .gitignore [hunk 3]
```diff
@@ -10,7 +10,9 @@ docs/schema.json
...
+videos/*
```
Comment (palisadoes) [inline .gitignore:14]: Ensure that:

1. any subdirectories in this directory also gets ignored
2. it only affects the directory in root

**Review by palisadoes — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Veer0x1) [reply .gitignore:15]: In the reference PR, Eshaan told to ignore this directory, but you say, should I ignore or not?

File: .gitignore (hunk 2 above)
Comment (Veer0x1) [reply .gitignore]: done

File: .gitignore (hunk 3 above)
Comment (Veer0x1) [reply .gitignore]: done

File: .gitignore (hunk 1 above)
Comment (palisadoes) [reply .gitignore:15]: You are moving image files that are referenced in the documentation markdown files and then ignoring them so that they don't get uploaded to the repository. You can see that they will be ignored by the gray icons

image

**Review by palisadoes — CHANGES_REQUESTED** [review]
See note

**Comment by Veer0x1** [comment]
> See note

Okay, I'm removing, `public/markdown/*` from `.gitignore`

**Review by palisadoes — APPROVED** [review]
